### ISSUE TITLE
Coronoid fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10799]
**Preferred term label:**
Coronoid fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
fracture of the coronoid process on the proximal ulna

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Radial head fracture
Monteggia fracture
Olecranon fracture
Distal humerus fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001134
- label: Coronoid fracture
- parents: [Fractured ulna (HP:0003987)](https://hpo.jax.org/app/browse/term/HP:0003987)
- pmid: PMID:39381286
- def: Fracture of the coronoid process on the proximal ulna.
- comment: The coronoid process acts to buttress against the posterior translation of the ulna on the humerus. It is also the attachment site for the anterior elbow capsule, brachialis, and the medial collateral ligament of the elbow. Coronoid fractures are critical injuries and are suggestive of elbow instability.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10799
- synonyms:
